utils/appliance/__init__.py

```diff
@@ -163,6 +163,15 @@ def get(self, cls, *args, **kwargs):
         assert 'appliance' not in kwargs
         return cls(appliance=self, *args, **kwargs)
 
+    def postgres_version(self):
```

Comment:
Make this a property using the `@property` decorator and you can drop the `()` from all the `.postgres_version()` calls and just use `.postgres_version` instead.